main.cc

```diff
@@ -1643,6 +1643,32 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 gossiper.local().unregister_(mm.local().shared_from_this()).get();
             });
 
+            utils::loading_cache_config perm_cache_config;
+            perm_cache_config.max_size = cfg->permissions_cache_max_entries();
+            perm_cache_config.expiry = std::chrono::milliseconds(cfg->permissions_validity_in_ms());
+            perm_cache_config.refresh = std::chrono::milliseconds(cfg->permissions_update_interval_in_ms());
+
+            auto start_auth_service = [&mm] (sharded<auth::service>& auth_service, std::any& stop_auth_service) {
+                supervisor::notify("starting auth service");
```

Comment:
Won't this mean we'll have two "starting auth service" messages, when you also use it later for the second "dummy" auth service?

I think we should include some context into the message so we distinguish the step of starting regular auth service from the maintenance-socket auth service.

Same for the `defer_verbose_shutdown` call below, the string should probably be different depending on the context.